src/index.ts

```diff
@@ -18,12 +18,18 @@ const mergePdfBuffers = (pdfBuffers: Array<Buffer>) => {
   return outStream.toBuffer();
 };
 
+const safeUrlAppend = (origin: String, path: String, file: String) => {
+  return origin + (path.length > 0 ? '/' : '') + 
+    path.substring(path.startsWith('/') ? 1 : 0, path.length - (path.endsWith('/') ? 1 : 0)) + 
+    '/' + file;
+}
```

Comment:
Is just append `baseUrl` to `origin` is not safe or enough ?

something like...
```js
basePath = baseUrl ? origin/baseUrl : origin ;
```

This part looks difficult for me 😢

Reply:
This is doing checking to prevent things like `origin//baseUrl` or `originbaseUrl`, because someone might supply `/baseUrl/` or 'baseUrl` as the argument. I would've used os.path.join but that would cause invalid urls on windows systems (`\` instead of `/`)

Reply:
> someone might supply /baseUrl/

Oh I understand 👍 Thanks!